libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -308,7 +343,6 @@ const ContestTitle = styled.div`
   display: flex;
   flex-wrap: wrap;
   gap: 0.25em;
-  font-size: 1.125em;
```

Comment:
Was this being overridden or otherwise unused?

Reply:
Ah, forgot to call this out - decided to reduce the font size of the contest titles to match the selections, to create a bit more room on the ballot. With the extra `~3pts` we're applying, they were starting to take up too much space and making it tough to fit enough contests at the default size